java/fury-core/src/test/java/org/apache/fury/serializer/PrimitiveSerializersTest.java

```diff
@@ -54,4 +59,62 @@ public void testUint16Serializer() {
     assertThrows(IllegalArgumentException.class, () -> serializer.xwrite(buffer, -1));
     assertThrows(IllegalArgumentException.class, () -> serializer.xwrite(buffer, 65536));
   }
+
+  @Data
+  @AllArgsConstructor
+  public static class PrimitiveStruct {
```

Comment:
Should be private.
```java
private static class PrimitiveStruct
```

Reply:
Private will work here too, but the generate code would be different. Codegen work best for public classes